### ISSUE TITLE
[Web] [Accessibility][11yMAS][Screen Readers][MAS1.4.3] [Luminosity ratio is less than 4.5:1 for the search place holder text .]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#696]
**User Experience:**
User with partial vision disability may not be able to clearly view the control or the text as the luminosity ratio is less than 4.5:1.

**Test Environment:**
OS Build: Mac OS Sierra 0.12.6
Browser: Safari 11.0.1
Tools Used: Colour Contrast Analyser
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0

**Attachments:**
![mas1 4 3 search_placeholder text](https://user-images.githubusercontent.com/33054846/32441565-3ca70420-c31e-11e7-989e-d2a1dd16f02b.png)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to Search Edit field and verify the luminosity ratio for the search Place holder text.

 **Current Behavior:** 
Luminosity ratio for the search place holder text is 4.0:1 .

**Expected Behavior:**
Luminosity ratio for the search place holder text should be equal or more than 4.5:1

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.4.3
